src/guides/v2.3/graphql/queries/index.md

```diff
@@ -565,3 +565,13 @@ query IntrospectionQuery {
   }
 }
 ```
+
+### Disabling introspection querying
+
+Introspection querying is enabled by default. If you wish to disable it in production mode to improve security, please add the following to env.php.
```

Comment:
```suggestion
Introspection querying is enabled by default. To disable it in production mode to improve security, add the following to your `env.php` file.
```

Reply:
I would strongly recommend specifying the full path to the configuration file. Very often only the filename brings a lot of confusion (happened to me as well in the documentation for 3rd party API system). There's a chance that a developer with no experience will create `env.php` file in the Magento root directory.

```suggestion
Introspection querying is enabled by default. If you wish to disable it in production mode to improve security, please add the following to `app/etc/env.php`.
```

Hope it makes sense

Reply:
@rogyar, good catch! I have corrected the path, hopefully it will add to the clarity!